### DISCUSSION TITLE
Items, Pickups and Powerups.

### DISCUSSION THREAD

**Discussion by oppotaku** [ikemen-engine/Ikemen-GO#895] [Feature Request] [answered]
Add Items, Pickups and Powerups, that give the player a new move/ability, weapon or buff (or debuff). They can be spawned when you want (which one you want) in practice and are spawned randomly in stages if you enable the feature.

**Comment by Windblade-GR01** [comment, marked as answer]
This already could be done via helpers.
A game-wide implementation could be made via `attachedChar` or `commonStates`.

In the future I could make a example implementation of this if you think is a good idea.
